src/encoded/static/components/publication.js

```diff
@@ -90,7 +86,7 @@ const PublicationComponent = (props, reactContext) => {
                 <Citation {...props} />
             </div>
 
-            {context.abstract || context.data_used || (context.datasets && context.datasets.length) || (context.identifiers && context.identifiers.length) ?
+            {context.abstract || context.data_used || (context.identifiers && context.identifiers.length) ?
```

Comment:
Oh, could you change that to `context.identifiers.length > 0`? A lot of times ESLint plugin misses that, but our ESLint config likes integer comparisons to be against 0 instead of truthy.

Reply:
Done